apps-rendering/src/components/Footer/index.tsx

```diff
@@ -1,63 +1,25 @@
 // ----- Imports ----- //
 
-import type { SerializedStyles } from '@emotion/react';
-import { css } from '@emotion/react';
-import { background } from '@guardian/common-rendering/src/editorialPalette';
 import type { ArticleFormat } from '@guardian/libs';
-import {
-	breakpoints,
-	from,
-	neutral,
-	remSpace,
-	textSans,
-} from '@guardian/source-foundations';
-import FooterContent from 'components/FooterContent';
+import { ArticleDisplay } from '@guardian/libs';
 import type { FC } from 'react';
-import { darkModeCss } from 'styles';
+import DefaultFooter, { defaultStyles } from './Footer.defaults';
+import ImmersiveFooter from './ImmersiveFooter';
 
 // ----- Component ----- //
 
-const styles = (format: ArticleFormat): SerializedStyles => css`
-	border-width: 0 1px;
-	${textSans.small({ lineHeight: 'regular' })};
-	margin-left: 0;
-	margin-right: 0;
-	padding-left: ${remSpace[3]};
-	padding-right: ${remSpace[3]};
-	padding-top: ${remSpace[4]};
-	padding-bottom: ${remSpace[4]};
-
-	${from.wide} {
-		margin: 0 auto;
-		width: ${breakpoints.wide}px;
-	}
-
-	a {
-		${textSans.small({ lineHeight: 'regular' })};
-		color: ${neutral[7]};
-		text-decoration: underline;
-	}
-
-	${darkModeCss`
-        color: ${neutral[60]};
-		background-color: ${background.articleContentDark(format)};
-
-        a {
-            color: ${neutral[60]};
-        }
-    `}
-`;
-
 interface Props {
-	isCcpa: boolean;
 	format: ArticleFormat;
+	isCcpa: boolean;
```

Comment:
nothing to change here, but what is `ccpa`?

Reply:
https://www.theguardian.com/us-news/2019/dec/30/california-consumer-privacy-act-what-does-it-do

Reply:
I've added some screenshots above showing what this new footer looks like with different values of `isCcpa`.